Content.Client/Preferences/UI/HumanoidProfileEditor.xaml.cs

```diff
@@ -445,14 +452,129 @@ public sealed partial class HumanoidProfileEditor : Control
 
             #endregion
 
+            #region Loadouts
+
+            _tabContainer.SetTabTitle(4, Loc.GetString("humanoid-profile-editor-loadouts-tab"));
+            _loadoutPreferences = new List<LoadoutPreferenceSelector>();
+            var loadouts = prototypeManager.EnumeratePrototypes<LoadoutPrototype>().OrderBy(l => l.ID).ToList();
+
+            var loadoutsEnabled = _configurationManager.GetCVar(CCVars.GameLoadoutsEnabled);
+            _tabContainer.SetTabVisible(4, loadoutsEnabled);
+            ShowLoadouts.Visible = loadoutsEnabled;
+            _configurationManager.OnValueChanged(CCVars.GameLoadoutsEnabled, enabled =>
+            {
+                _tabContainer.SetTabVisible(4, enabled);
+                ShowLoadouts.Visible = enabled;
+            });
```

Comment:
Isn't doing anonymous subs like this going to leak every single time the lobby opens

Reply:
I have no idea